careerfairsystems/arkad-app-2018
# Add Fabric

## Change description

Adds Fabric with Crashlytics to the project.

## How to verify

Confirm that errors are logged in Crashlytics. Not really sure if you can test this before release. But there are some info on how to enforce crashes here: https://github.com/corymsmith/react-native-fabric#crashlytics-usage.

## Issues fixed

This fixes #99.
